walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -42,7 +43,9 @@
   private static final int TIME_0930AM = TimeRange.getTimeInMinutes(9, 30);
   private static final int TIME_1000AM = TimeRange.getTimeInMinutes(10, 0);
   private static final int TIME_1100AM = TimeRange.getTimeInMinutes(11, 00);
+  private static final int TIME_1230PM = TimeRange.getTimeInMinutes(12, 30);
 
+  private static final int DURATION_15_MINUTES = 15;
```

Comment:
IMO: this file abuses constants in order to have nice "fluent" tests, and the payoff isn't worth the cost (of having to reference the value occasionally).

I don't recommend changing anything though, just wanted to suggest that you not try to make literally everything in a test a constant like this test does when you're writing your own tests. It's OK for tests to have code duplication and be slightly verbose because all of the behavior in the test should be clear and unambiguous.

Reply:
Ack